script/test

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# script/test: Run test suite for application. Optionally pass in a path to an
+#              individual test file to run a single test.
```

Comment:
Is this true? I don't think it's currently set up to handle individual files

Reply:
Thank you for spotting this, thats a line i copied from the script/ templates and didn't come back to. I can set this up fairly easily. But wondering if setting it up will become an issue if we add other test suites to this script? 

e.g. @snim2 I know you have been looking at adding html-proofer to a test suite, which I think doesn't accept file paths as far as I can see? Will it be confusing if arguments can be passed in which are only relevant for different sections of the test script? 

I wonder if its simpler to keep a standard script for the whole suite, and developers have the tools to run certain commands individually using this as a reference.

Reply:
It's totally fine to have arguments that are used in some places but not others. We usually add both testing and linting to these scripts, so it's quite common.

You may already have [read this](https://github.com/github/scripts-to-rule-them-all), but if not it's a good introduction to this design pattern.

Reply:
Thanks Sarah. I have updated this now, and it can also accept standard flags, such as --fix, which is explained in the if/else.

Reply:
OK, but does the comment at the top of the file match the behaviour?

Reply:
Yes, you can now pass in one file or a directory - which works for this implementation but we may need to keep an eye on if adding test suites or proofing to the script. Have kept specifics of standards arguments to the section where linting is run, but let me know if you think it would be better to move that to the top